pennylane/ops/qubit/observables.py

```diff
@@ -91,6 +91,9 @@ def __init__(self, A: TensorLike, wires: WiresLike, id: Optional[str] = None):
     @staticmethod
     def _validate_input(A: TensorLike, expected_mx_shape: Optional[int] = None):
         """Validate the input matrix."""
+        if qml.math.is_abstract(A):
+            return
```

Comment:
Why this is needed now?

Reply:
At some point when the qnode is executed `Hermitian.compute_matrix` is called. The `_validate_input` function to determine if the input is Hermitian is not jittable, so we check if the input is abstract and skip the validation if it is.